sunpy/visualization/__init__.py

```diff
@@ -1,3 +1,6 @@
 # -*- coding: utf-8 -*-
 
 from sunpy.visualization.visualization import *
+from sunpy.visualization.imageanimator import *
```

Comment:
why? add these?

Reply:
just incase.

Reply:
:-1: We shouldn't mess with the API unless we need to.

Reply:
FINE